src/auro-popover.js

```diff
@@ -102,8 +102,14 @@ class AuroPopover extends LitElement {
     if (this.sticky) {
       this.trigger.addEventListener('click', handleShow);
     } else {
-      this.trigger.addEventListener('mouseenter', handleShow);
-      this.trigger.addEventListener('mouseleave', handleHide);
+
+      if(this.trigger.parentElement.nodeName === 'AURO-POPOVER') {
+        this.addEventListener('mouseenter', handleShow);
+        this.addEventListener('mouseleave', handleHide);
+      } else {
+        this.trigger.addEventListener('mouseenter', handleShow);
+        this.trigger.addEventListener('mouseleave', handleHide);
+      }
```

Comment:
```suggestion
      const element = this.trigger.parentElement.nodeName === 'AURO-POPOVER' ? this : this.trigger;
      element.addEventListener('mouseenter', handleShow);
      element.addEventListener('mouseleave', handleHide);
```

Reply:
I like it - nice, clean adjustment @geoffrich.